src/enclave/main.cpp

```diff
@@ -172,13 +180,6 @@ extern "C"
         return CreateNodeStatus::MemoryNotOutsideEnclave;
       }
 
-      if (!ccf::pal::is_outside_enclave(
-            ec.from_enclave_buffer_start, ec.from_enclave_buffer_size))
-      {
-        LOG_FAIL_FMT("Memory outside enclave: from_enclave buffer start");
-        return CreateNodeStatus::MemoryNotOutsideEnclave;
-      }
-
       if (!ccf::pal::is_outside_enclave(
```

Comment:
I think we need to move all of these checks earlier, include the offsets.

Reply:
Yes of course - thank you!